Tasks/Common/MSBuildHelpers/ArgumentFunctions.ps1

```diff
@@ -24,9 +24,31 @@ function Format-MSBuildArguments {
         if ($MaximumCpuCount) {
             $MSBuildArguments = "$MSBuildArguments /m"
         }
+        
+        $userAgent = Get-UserAgentString
```

Comment:
Can you call `Get-VstsTaskVariable -Name AZURE_HTTP_USER_AGENT` instead of adding a function for this one-liner? Tracing is built in to the Get-VstsTaskVariable command, so it will actually be better than calling the function.